packages/flutter/lib/src/widgets/spell_check.dart

```diff
@@ -2,11 +2,25 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-import 'package:flutter/foundation.dart';
 import 'package:flutter/painting.dart';
 import 'package:flutter/services.dart'
     show SpellCheckResults, SpellCheckService, SuggestionSpan, TextEditingValue;
 
+import 'editable_text.dart' show EditableTextState;
+import 'framework.dart' show BuildContext, Widget, immutable;
+
+/// Signature for a function that builds a widget to use as the spell check
+/// suggestions toolbar for [EditableText].
+///
+/// See also:
+///
+///  * [SpellCheckConfiguration], where this builder can be set to use by
+///    default for displaying spell check suggestions for misspelled words.
+typedef SpellCheckSuggestionsToolbarBuilder = Widget Function(
+  BuildContext context,
+  EditableTextState editableTextState,
+);
```

Comment:
Nit: This typedef is the same as EditableTextContextMenuBuilder.  Maybe just use that instead of creating a new one?